mkt/extensions/models.py

```diff
@@ -23,14 +24,6 @@
 log = commonware.log.getLogger('z.extensions')
 
 
-def _make_language_choices(languages):
-    return [(to_language(lang_code), lang_name)
-            for lang_code, lang_name in languages.items()]
-
-
-LANGUAGE_CHOICES = lazy(_make_language_choices, list)(settings.LANGUAGES)
```

Comment:
Not needed because I relaxed the `default_language` field, we don't actually need to limit choices in there.